src/main/vue/src/store/sound/mutations.ts

```diff
@@ -0,0 +1,14 @@
+import { MutationTree } from 'vuex';
+import { SoundState, Sound } from './types';
+
+export const mutations: MutationTree<SoundState> = {
+    soundChosen(state, sound: Sound) {
```

Comment:
shouldn't be names of mutations verbs in imperative rather that participles (`chooseSound` instead of `soundChosen`)?

Reply:
Right, will change

Reply:
On second thought, no. Actions are the ones which should be expressed in the imperative form. Mutations are the ones you commit. "Commit chosen sound" sounds much better than "commit choose sound".